### PR TITLE
merge 

### PR DESCRIPTION
**Pull request by allendm-msft**

### CODE REVIEW

**Comment by kaadhina** [comment]
Looks fine. We might need to make this change in ts implementation too.

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/DeployVisualStudioTestAgent/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 36
+        "Patch": 38
```
Comment (SatishThareja) [inline Tasks/DeployVisualStudioTestAgent/task.json]: why bumping up the version by 2 ? Same comment for RunDistributed task as well.

Comment (kaadhina) [inline Tasks/DeployVisualStudioTestAgent/task.json]: So these versions are now one above the version in releases/M105.

**Comment by kaadhina** [comment]
Looks fine

File: Tasks/DeployVisualStudioTestAgent/task.json (hunk 1 above)
Comment (allendm-msft) [inline Tasks/DeployVisualStudioTestAgent/task.json]: @SatishThareja - @kaadhina has it right. if anyone from a rtm agent upgrades they should get the new task thats why the bump up higher than m105

**Pull request merged**